qgis2web/leafletStyleScripts.py

```diff
@@ -167,7 +169,7 @@ def getLayerStyle(layer, sln, interactivity, markerFolder,
 
 
 def getSymbolAsStyle(symbol, markerFolder, layer_transparency, interactivity,
-                     sln, sl, useMapUnits, feedback):
+                     sln, sl, useMapUnits, feedback, label=''):
```

Comment:
I had to add a `label` param to `getSymbolAsStyle()` so that the category label could be used in the derivative SVG file name.